bigtable/google/cloud/bigtable/table.py

```diff
@@ -276,6 +284,31 @@ def read_rows(self, start_key=None, end_key=None, limit=None,
         # We expect an iterator of `data_messages_v2_pb2.ReadRowsResponse`
         return PartialRowsData(response_iterator)
 
+    def mutate_rows(self, rows):
+        """Mutates multiple rows in bulk.
+    
+        :type rows: list
+        :param rows: List or other iterable of :class:`.DirectRow` instances.
+        
+        :rtype: list
+        :returns: A list of tuples (``MutateRowsResponse.Entry`` protobuf 
+                  corresponding to the errors, :class:`.DirectRow`) 
+        """
+        _check_rows(self.name, rows)
+        mutate_rows_request = _mutate_rows_request(self.name, rows)
+
+        unsuccessfully_mutated_rows = []
+        client = self._instance._client
+        responses = client._data_stub.MutateRows(mutate_rows_request)
+        for response in responses:
+            for entry in response.entries:
+                if not entry.status.code:
+                    rows[entry.index].clear()
```

Comment:
Why is it necessary to clear the successfully mutated `row`? If it **is** necessary, you should definitely mention it in the docstring.